Dockerfile-debian-otel.template

```diff
@@ -0,0 +1,97 @@
+ARG IMAGE=nginxinc/nginx-unprivileged:%%NGINX_VERSION%%
+FROM $IMAGE
+
+ENV OTEL_VERSION=%%OTEL_VERSION%%
+
+ARG UID=101
+ARG GID=101
+
+USER root
+
+RUN set -x; \
+    NGINX_GPGKEY_PATH=/etc/apt/keyrings/nginx-archive-keyring.gpg; \
+    dpkgArch="$(dpkg --print-architecture)" \
+    && nginxPackages="%%PACKAGES%%
+    " \
+    && case "$dpkgArch" in \
+        amd64|arm64) \
+# arches officialy built by upstream
```

Comment:
Since I saw you fixed a typo somewhere else, can we also fix this typo in the core templates (anything that ends in .template)? Running `update.sh` should then propagate the fix everywhere else.